packages/node_modules/nav-frontend-veileder-style/src/index.less

```diff
@@ -1,35 +1,35 @@
 @import '@{coreModulePath}nav-frontend-core/less/_variabler';
 
-.nav-frontend-veileder {
+.veileder {
```

Comment:
Litt for generisk klassenavn kanskje, føler dette fort vil ha kollisjoner med ting folk allerede har i koden sin.

Reply:
Hm, mulig. Skal jeg bare bruke `nav-frontend-veileder` likevel?